cas-server-core-api-ticket/src/main/java/org/jasig/cas/ticket/ExpirationPolicy.java

```diff
@@ -24,4 +24,20 @@
      * @return true if the ticket is expired, false otherwise.
      */
     boolean isExpired(TicketState ticketState);
+
+    /**
+     * Describes the time duration where this policy should consider the item alive.
+     * Once this time passes, the item is considered expired and dead.
+     * @return alive time in seconds. A zero value indicates the time duration
+     * is not supported or is inactive.
+     */
+    Long getTimeToLive();
```

Comment:
In theory, it's possible yes but that would of course be met with an error. Per the contract, null values are not permitted. 

Trying to remember: I think the rationale was 1) I generally prefer objects over primitives 2) certain registries support in, and some support long. Rather then returning a primitive and casting, etc I simply wanted to do `getTimeToLive().intValue()` when an int was required. 

This might of course be problematic; the deployer would have to be aware of what the registry supports. If you choose your expiration policy values for a ticket to last as long as a Long can, the registry would certainly bomb out because it can only support int max values. But, nothing we can do about that.